src/components/ui/VoiceChat.tsx

```diff
@@ -248,6 +248,7 @@ function VoiceChat() {
       document.getElementById("form").style.display = "block";
       //remove channel control buttons
       document.getElementById("room-header").style.display = "none";
+      document.getElementById("room-footer").style.display = "none";
```

Comment:
code smell: put these things `document.getElementById("...")` always in variables. becuase if string name changes, you only have to edit those in one place. 

rule of thumb is always: these hard coded values should be saved in a variable. again: if something changes, you only have to change one place, because the variable name does not really change.

Reply:
thanks for noticing, I quickly replaced all and tested it